library/cwm/src/lib/cwm/replace_point.rb

```diff
@@ -20,7 +20,16 @@ def initialize(id: "_placeholder", widget: Empty.new("_initial_placeholder"))
 
     # @return [UITerm]
     def contents
-      ReplacePoint(Id(widget_id), @widget)
+      # Use for contents empty widget, because if @widget passed to constructor
+      # will be used then CWM itself will handle events for this initial widget.
+      # This is against replace point idea that replace point due to its dynamic
+      # content will handle events itself
+      ReplacePoint(Id(widget_id), Empty(Id("___cwm_rp_empty")))
```

Comment:
Ah, I think I understand now. Your English is so bad it takes me quite long to get what you mean:

```rb
# In `contents` we must use an Empty Term, otherwise CWMClass
# would see an {AbstractWidget} and handle events itself,
# bypassing our {#handle}
```

 (BTW I don't see why we need it but that's another topic)